### DISCUSSION TITLE
token di aggiornamento infiniti

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2580] [HUB: Home Assistant]
ciao a tutti, 
mi ritrovo una lista di token di aggiornamento infinita.
da quando avevo effettuato l'installazione li avevo ma parliamo di almeno un centinaio... come posso eliminarli?
ovviamente tramite procedura automatica e non dovendo cliccare su ogni singolo cestino!

**Comment by KC-inDomus** [comment]
ma dove, su cosa, in che ambiente, su quale configurazione.

**Reply by Matty1184** [reply]
impostazioni generali utente

Token di aggiornamento
Ogni token di aggiornamento rappresenta una sessione di accesso. I token di aggiornamento saranno rimossi automaticamente quando fai clic su Disconnetti. I seguenti token di aggiornamento sono attualmente attivi per il tuo account.

Token di aggiornamento per http://192.168.xxxxxxx:xxxxx/
Creato 3 ore fa
Ultimo utilizzo 20 minuti fa da 192.168.xxxxxxxxxxxx

come questi ne avrò un centinaio

**Reply by KC-inDomus** [reply]
di cosa stai parlando

**Reply by Matty1184** [reply]
sezione profilo utente HA (HA OS 10.4)
dopo la sezione password ci sono i token di lunga durata e i token di aggiormento, immagino per ogni dispositivo/accesso.
nella sezione token di aggiornamento ne ho tantissimi. a cosa sono dovuti?

**Reply by KC-inDomus** [reply]
non lo so. intanto sposto il thread nella sezione giusta

**Reply by 50494554524F** [reply]
ne crea uno ogni volta che fai login... clicca disconnetti e poi riloggati

**Reply by Matty1184** [reply]
già provato, mi cancella solo quello dell'ultima sessione, gli altri restano in essere.

**Reply by 50494554524F** [reply]
confermo il bug o il fatto che siano token associati ad altre sessioni browser / dispositivi / etc
